src/sfctl/tests/custom_help_text_correctness_test.py

```diff
@@ -0,0 +1,237 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
+
+"""Tests that the text provided as custom help text is always up tp date.
+Check the help text defined in the helps section appear in the service fabric Python SDK.
+This will check that custom help text is always up to date with those defined in
+Service Fabric Swagger specifications.
+
+This does not test for missing information. In the future, consider requiring long summaries
+for all descriptions."""
+
+from __future__ import print_function
+import unittest
+from os import listdir
+from os.path import dirname, join, abspath, pardir
+from imp import load_source
+
+CURRENT_DIR = dirname(abspath(__file__))  # This should be <location>/src/sfctl/tests
+SRC_PARENT_DIR = dirname(dirname(dirname(CURRENT_DIR)))
+CUSTOM_SDK_HELPER_FILE = join(SRC_PARENT_DIR, 'scripts', 'check_and_use_custom_sdk.py')
+
+load_source('custom_sdk_helper', CUSTOM_SDK_HELPER_FILE)
+
+import custom_sdk_helper  # pylint: disable=wrong-import-position,import-error
+
+
+class CustomHelpTextCorrectnessTests(unittest.TestCase):
+    """Tests that the text provided as custom help text is always up to date."""
+
+    # If a line from a customs helps file starts with the following, do not validate.
+    # This list should eventually become empty
+    exclusion_list = [
```

Comment:
How would this list eventually get smaller?

Reply:
Added more detailed explanation in comments in the file.